lib/builtin.pm

```diff
@@ -40,12 +40,9 @@ can be requested for convenience.
 Individual named functions can be imported by listing them as import
 parameters on the C<use> statement for this pragma.
 
-The overall C<builtin> mechanism, as well as every individual function it
-provides, are currently B<experimental>.
-
-B<Warning>:  At present, the entire C<builtin> namespace is experimental.
-Calling functions in it will trigger warnings of the C<experimental::builtin>
-category.
+B<Warning>:  At present, the many functions in the C<builtin> namespace are
```

Comment:
Should this not be "many of the functions"?

Reply:
Ahyes; fixed.